lib/modules/platform/gitlab/index.ts

```diff
@@ -852,13 +852,9 @@ export async function reattemptPlatformAutomerge({
   number: iid,
   platformOptions,
 }: ReattemptPlatformAutomergeConfig): Promise<void> {
-  try {
-    await tryPrAutomerge(iid, platformOptions);
+  await tryPrAutomerge(iid, platformOptions);
```

Comment:
@viceice, I couldn't see a way that `tryPrAutomerge` would throw for GitLab, so I removed the try/catch to resolve the coverage issue. Let me know if that's not okay!

Reply:
Also, raising this since I noticed, in case you all weren't already aware, it looks like Codecov has been broken since mid-February: https://app.codecov.io/github/renovatebot/renovate

Reply:
🤔

https://github.com/renovatebot/renovate/actions/runs/8308110045/job/22738028070

it says upload successfully.

@zharinov can you check?

Reply:
🤦‍♂️

Reply:
Hey @straub, good catch, our coverage is fixed now 👍